packages/marshal/test/test-marshal.js

```diff
@@ -287,21 +288,25 @@ test('records', t => {
       if (opt === 'enumStringData') {
         props.key1 = { enumerable: true, value: 'data' };
       } else if (opt === 'enumStringFunc') {
-        props.key2 = { enumerable: true, value: () => 0 };
-      } else if (opt === 'enumStringGet') {
-        props.key3 = { enumerable: true, get: () => 0 };
+        props.enumStringFunc = { enumerable: true, value: () => 0 };
```

Comment:
If you're going to give the string-named properties meaningful string names, which is nice, you could also give the symbols above similar meaningful names.

Reply:
done